mon-pix/app/templates/components/routes/register-form.hbs

```diff
@@ -79,7 +79,7 @@
         <div id="register-username-container">
           <FormTextfield @label="Mon identifiant"
                          @textfieldName="username"
-                         @inputBindingValue={{this.studentDependentUser.username}}
+                         @inputBindingValue={{this.username}}
```

Comment:
pourquoi remplacer `this.studentDependentUser.username` par `this.username` sur le username et pas sur l'email ? 🤔
Il y a anguille sous roche

Reply:
En fait le problème venait du fait qu'on reset des l'email ou username avant de faire appel à l'api. Par contre en cas d'erreur, on perd les valeurs précedentes. 
Pour l'email, il n y a pas de valeur champ email, je suis en train de restaurer les values réinitialiser dans le catch en cas d'erreur pour les deux  :)

Reply:
Est-ce que le problème ne vient pas du fait qu'on fait (si j'ai bien compris) du _binding_ des champs directement sur les modèles qui vont être soumis à l'API, au lieu de construire le bon modèle sur la base des champs de formulaire juste au moment de l'appel ?

Reply:
Effectivement, c'est la bonne solution. Avoir des inputs de login dédié ihm et faire un mapping avec le model d'api lors de l'appel. C'est fixed